components/dashboard.js

```diff
@@ -1,19 +1,17 @@
 import Head from 'next/head'
 import styled, { injectGlobal, ThemeProvider } from 'styled-components'
-import { normalize, padding } from 'polished'
+import { normalize } from 'polished'
 
 const Container = styled.main`
-  ${padding('4.25em', null, '4.25em', null)}
-  align-content: center;
   align-items: center;
   background-color: ${props => props.theme.palette.backgroundColor};
   color: ${props => props.theme.palette.textColor};
-  display: grid;
-  grid-gap: 2em;
-  grid-template-columns: repeat(3, 1fr);
-  height: 100vh;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
   justify-content: center;
-  justify-items: center;
+  min-height: 100vh;
```

Comment:
I think `min-height: 100vh` is corrent, because we need the full height of the screen for the container.